suite-native/module-home/src/screens/HomeScreen/components/StakingBanner.tsx

```diff
@@ -0,0 +1,185 @@
+import { useCallback, useMemo, useState } from 'react';
+import { TouchableOpacity } from 'react-native';
+import Animated, { FadeInDown, LinearTransition } from 'react-native-reanimated';
+import { useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { useSelectorDeepComparison } from '@suite-common/redux-utils';
+import { NetworkSymbol, getNetworkDisplaySymbol } from '@suite-common/wallet-config';
+import {
+    StakeRootState,
+    selectHasRunningDiscovery,
+    selectIsDeviceAuthorized,
+    selectPoolStatsApyData,
+} from '@suite-common/wallet-core';
+import { Account } from '@suite-common/wallet-types';
+import {
+    AssetsRootState,
+    selectDeviceNetworksWithAssets,
+    selectVisibleDeviceAccountsKeysByNetworkSymbol,
+} from '@suite-native/assets/src/assetsSelectors';
+import {
+    AnimatedCard,
+    Box,
+    HStack,
+    Text,
+    VStack,
+    cardVariantToColorsMap,
+} from '@suite-native/atoms';
+import { CryptoIcon, Icon } from '@suite-native/icons';
+import { Translation } from '@suite-native/intl';
+import { selectIsStakingEnabled } from '@suite-native/module-trading';
+import {
+    AppTabsParamList,
+    AppTabsRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    TabToStackCompositeNavigationProp,
+} from '@suite-native/navigation';
+import { selectDeviceAccountsWithStaking } from '@suite-native/staking';
+import { TokensRootState, selectHasDeviceAnyTokensForNetwork } from '@suite-native/tokens';
+
+import { StakingBannerBottomSheet } from './StakingBannerBottomSheet';
+
+type NavigationProp = TabToStackCompositeNavigationProp<
+    AppTabsParamList,
+    AppTabsRoutes.HomeStack,
+    RootStackParamList
+>;
+
+const STAKING_BANNER_NETWORK_SYMBOL: NetworkSymbol = 'sol';
+
+export const StakingBanner = () => {
+    const navigation = useNavigation<NavigationProp>();
+
+    const IsStakingEnabled = useSelector(selectIsStakingEnabled);
```

Comment:
```suggestion
    const isStakingEnabled = useSelector(selectIsStakingEnabled);
```

or maybe useFeatureFlag hook might be enough